src/library/CMakeLists.txt

```diff
@@ -26,47 +26,83 @@
 #  POSSIBILITY OF SUCH DAMAGE.
 #
 
-add_library(commissioner STATIC
-    openthread/bloom_filter.cpp
-    openthread/crc16.cpp
-    openthread/pbkdf2_cmac.cpp
-    openthread/random.cpp
-    openthread/sha256.cpp
-    cbor.cpp
-    coap.cpp
-    commissioner_impl.cpp
-    commissioner_safe.cpp
-    commissioning_session.cpp
-    cose.cpp
-    dtls.cpp
-    error.cpp
-    logging.cpp
-    multicast_dns.cpp
-    network_data.cpp
-    socket.cpp
-    tlv.cpp
-    token_manager.cpp
-    udp_proxy.cpp)
-
-target_link_libraries(commissioner PRIVATE
-    cose
-    mdns
-    mbedtls
-    pthread
-    fmt::fmt
-    event_core_static
-    event_pthreads_static
-    commissioner-common)
+add_library(commissioner)
+target_sources(
+  commissioner
```

Comment:
I understand that CMake convention is to use 2 spaces, but openthread uses 4 spaces for indent everywhere and I'd like to keep consistent with openthread. @gocarlos Do you mind to keep indent as 4 spaces?

Reply:
Always funny to have intendation discussions in MRs ;)

What about introducing this
* https://github.com/cheshirekow/cmake_format
* https://marketplace.visualstudio.com/items?itemName=cheshirekow.cmake-format

Reply:
It looks good to me. We can consider apply this tool to other openthread projects. For example, https://github.com/openthread/openthread.